github_api/voting.py

```diff
@@ -137,6 +138,9 @@ def get_vote_sum(api, votes):
     negative = 0
     for user, vote in votes.items():
         weight = get_vote_weight(api, user)
+        if random.randint(0,500) == 42:
+            #Jackpot!
+            weight = weight * 100
```

Comment:
Correct me if I'm wrong, but isn't this recalculated every time the votes are polled anyway?